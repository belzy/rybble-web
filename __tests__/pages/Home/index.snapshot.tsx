import { render } from '@testing-library/react';
import Home from '../../../pages/index';

describe('Home snapshot', () => {

  it ('should render correctly', () => {
    
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();

  });

});