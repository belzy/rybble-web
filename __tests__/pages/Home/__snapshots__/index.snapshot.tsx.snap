// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot should render correctly 1`] = `
<div>
  <h1>
    Welcome to Next.js
  </h1>
</div>
`;
